### ISSUE TITLE
Dialogs are unreadable with Windows screen scaling!

### ISSUE THREAD

**Issue by LorenAmelang** [gvellut/FreehandRasterGeoreferencer#58]
I'm using a Surface Book with a 3000x2000 pixel screen that is scaled to 1500x1000 by Windows 10 screen scaling. QGIS mostly works well. Some dialogs open in tiny windows, but I can manually drag those to useful sizes, and the text contents are always scaled properly. 

Here's what I see from FreehandRasterGeoreferencer - IF I'm able to drag the dialog windows larger: 

![2022-08-27 P](https://user-images.githubusercontent.com/1297090/187054204-c319e94c-79df-4d0e-876c-3300bdd1afe6.png)

The "not found" dialog opens tight around the small text area. It takes about a dozen tries to grab exactly in the right place to enlarge the dialog, and the text remains unchanged and half cut off even with the outline bigger. The error dialog opens with properly sized but mostly hidden text, which is at least readable once I drag the outline larger. 

The real problem is that the missing file name is hidden under the folder location! When I open a project and FreehandRasterGeoreferencer puts up a series of errors, I can't see which specific file it is complaining about each time. If I choose wrong, it reprojects some other layer to some wild position and I have to start over. 

Actually I'm not sure why it finds those errors. Maybe if a project is moved to a new drive or folder? I totally believe "The plugin result is "virtual". Only developers of QGis know how georeferencer affects such "non-system" objects."

**Comment by gvellut** [comment]
Thank you for your feedback.

There is indeed a problem with scaling (at least on Windows). I also noticed the issue when a I used the plugin a few months ago but I wasn't sure of the cause. Thanks to your report, it makes more sense. I will have a look at it.

> Actually I'm not sure why it finds those errors. Maybe if a project is moved to a new drive or folder? I totally believe "The plugin result is "virtual". Only developers of QGis know how georeferencer affects such "non-system" objects."

What do you mean by that? Unless the images are in a folder directly below the one containing the .qgs or .qgz, the paths are saved as absolute inside the project file. So if you move the project to another machine, there may be an issue with the path. You can check inside the .qgs (or unzip the .qgz and open the .qgs inside) to see if the paths to the images make sense.

**Comment by LorenAmelang** [comment]
Yes, the two different dialogs in my screenshot are classic examples of different Windows scaling problems.

I believe all of my images are in the same folder with the .qgz. Digging in the .qgs, I see some curiosities...


Most layers show: source="./<layer name>

`    <layer-tree-layer patch_size="-1,-1" providerKey="gdal" source="./WLR all parcels (assessor) (pdf copy) (gref).jpg" name="WLR all parcels (assessor) (pdf copy) (gref)" expanded="0" id="WLR_all_parcels__assessor___pdf_copy___gref__fc65d839_2b95_4192_8ec8_013cbcdb2aa1" checked="Qt::Unchecked" legend_split_behavior="0" legend_exp="">
`

But a few, the problem ones, show: source="" and "FreehandRasterGeoreferencerLayerProvider"

`    <layer-tree-layer patch_size="-1,-1" providerKey="FreehandRasterGeoreferencerLayerProvider" source="" name="Walker Lake (our parcel) Survey - C2 D60 P100 (squared)" expanded="1" id="Walker_Lake__our_parcel__Survey___C2_D60_P100__squared__4f78bb11_fd5c_4318_9400_e74f5912be93" checked="Qt::Unchecked" legend_split_behavior="0" legend_exp="">
`
 
One (an xyz layer) shows a full path, but to the current proper location:

`   <layer-tree-layer patch_size="-1,-1" providerKey="gdal" source="GPKG:C:/Users/loren/Giant Files/QGIS files/Projects - WLR/QGIS only/WLR fresh 220823/Google Hyb (clip to project) (2 meter).gpkg:Google Hyb (clip to project) (2 meter)" name="Google Hyb (clip to project) (2 meter)" expanded="0" id="Google_Hyb__clip_to_project___2_meter__3a461e26_fa1f_4808_bb7e_de127194caab" checked="Qt::Unchecked" legend_split_behavior="0" legend_exp="">
`
 
But most are in ./ locations (I thought this was done identically to the previous one): 

`    <layer-tree-layer patch_size="-1,-1" providerKey="gdal" source="./Google Hyb (clip to project) (1 meter).gpkg" name="Google Hyb (clip to project) (1 meter)" expanded="0" id="Google_Hyb__clip_to_project___1_meter__df429d8d_5439_4187_ae60_68e423c6e8ca" checked="Qt::Unchecked" legend_split_behavior="0" legend_exp="">
`

Here's one you georeferenced:

```
    <layer-tree-layer patch_size="-1,-1" providerKey="FreehandRasterGeoreferencerLayerProvider" source="" name="E GH Grading snip" expanded="1" id="E_GH_Grading_snip_c724bef1_33f7_4687_aa53_bcf5a60e1823" checked="Qt::Checked" legend_split_behavior="0" legend_exp="">
      <customproperties>
        <Option/>
      </customproperties>
    </layer-tree-layer>
...
      <customproperties>
        <Option type="Map">
          <Option type="QString" name="blendMode" value="SourceOver"/>
          <Option type="QString" name="filepath" value="C:/Users/loren/Giant Files/QGIS files/Projects - WLR/QGIS only/WLR fresh 220823/E GH Grading snip.PNG"/>
          <Option type="double" name="rotation" value="2.758"/>
          <Option type="QString" name="title" value="E GH Grading snip"/>
          <Option type="int" name="transparency" value="30"/>
          <Option type="double" name="xCenter" value="-13732686.37764909"/>
          <Option type="double" name="xScale" value="0.08574314653330291"/>
          <Option type="double" name="yCenter" value="4770539.187957743"/>
          <Option type="double" name="yScale" value="0.08574314653330291"/>
        </Option>
      </customproperties>

```
Combines source="" with correct full path later...  

None show any of the previous folder locations. 

Attaching the file - maybe you will understand it better...

[WLR fresh 220823.zip](https://github.com/gvellut/FreehandRasterGeoreferencer/files/9440389/WLR.fresh.220823.zip)

Thanks for the clues!
